package.json

```diff
@@ -78,6 +81,7 @@
     "jest:debug": "node --inspect-brk ./node_modules/.bin/jest --runInBand --env=jsdom",
     "test": "npm run jest",
     "test:debug": "npm run jest:debug",
+    "test:e2e": "npm run e2e",
```

Comment:
It might make sense to also introduce a task starting e2e tests inside docker container if you only need to verify that your change in UI doesn't break the test and you don't need to see what is happening.

Reply:
I could add this.
We have several people using mac, I being one. I get the following error when running `npm run image:start`:
```
The path /etc/localtime
is not shared from OS X and is not known to Docker
```

Reply:
Please report this kind of issues in https://github.com/fabric8io/fabric8-test/issues/

Reply:
Reported: https://github.com/fabric8io/fabric8-test/issues/1123